pysat/_constellation.py

```diff
@@ -745,7 +745,7 @@ def to_inst(self, common_coord=True, fill_method=None):
             # Get the common coordinates needed for all data
             for cinst in self.instruments:
                 if not cinst.pandas_format:
-                    for new_coord in cinst.data.dims.keys():
+                    for new_coord in cinst.data.sizes.keys():
```

Comment:
How far back does .sizes work?

Reply:
Currently passing tests in the operational branch for:
xarray-0.16.2